composer.json

```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=7.1",
         "symfony/symfony": "^3.0",
-        "doctrine/orm": "~2.5.4",
+        "doctrine/orm": "~2.6.6",
```

Comment:
Can you make sure that `~2.5.4` is also still allowed here? Maybe rather set it to `~2.5` instead